_sources/overview.rst

```diff
@@ -2373,7 +2538,7 @@ You can find more complete documentation of the directives on our `Runestone doc
 Now that you have seen some of these tools in action, you can do more exploration by going to `Runestone library <http://runestoneinteractive.org/library.html>`_
 and browsing through some of our existing books.
 
-We hope you will find these tools and materials useful.  If you want to get more involved, feel free to download the tools and write your own materials.  Everything you need, instructions and help for contributing or writing your own materials, and links to our GitHub repositories, can be found `here <http://runestoneinteractive.org/build/html/index.html>`_
+We hope you will find these tools and materials useful.  If you want to get more involved, feel free to download the tools and write your own materials.  Everything you need, instructions and help for contributing or writing your own materials, and links to our GitHub repositories, can be found `here <http://runestoneinteractive.org/build/html/index.html>`__.
```

Comment:
Why the double `_` The single works fine?

Reply:
To avoid an error -- Sphinx complains that `here` is defined twice. The double underscore makes it anonymous, avoid this.